### PR TITLE
WIP: remove .internal dns lookups from `fly pg attach`

### PR DESCRIPTION
**Pull request by tvdfly**
- Update agentclient.Instances() to concurrently query gql with original `instances <org> <app>` call to agent
- If we get different answers back, use the gql answers
- Report discrepancies and errors to sentry; if we get any results use those (i.e., if gql fails, but dns lookup succeeds use dns results)
- Update the pg commands to call agentclient to get list of instances
- All the pg commands now connect to the cluster leader's ip

### TESTING:

for a machines pg cluster:
- [x] ~~`fly launch` with pg provision~~ (n/a rn we make nomad pg clusters from fly launch)
- [x] `fly pg connect`
- [x] `fly pg attach`
- [x] `fly pg detach`
- [x] `fly pg config view`
- [x] `fly pg config update`
- [x] `fly pg db`
- [x] `fly pg failover`
- [x] `fly pg restart`
- [x] `fly pg users`

rinse and repeat for a nomad pg cluster
- [x] `fly launch` with pg provision
- [x] `fly pg connect`
- [x] `fly pg attach`
- [x] `fly pg detach`
- [x] `fly pg config view`
- [x] `fly pg config update`
- [x] `fly pg db`
- [x] ~~`fly pg failover`~~ (not supported for nomad)
- [x] `fly pg restart`
- [x] `fly pg users`

test existing callers of agentclient.Instances() to ensure I didn't regress anything:
- [x] machines app: `fly ssh console --select`
- [x] machines app: `fly proxy 3666:80 --select`
- [x] nomad app: `fly ssh console --select`
- [x] nomad app: `fly proxy 3666:80 --select`

other:
- [x] testflight run

### CODE REVIEW

File: internal/command/dig/dig.go [hunk 1]
```diff
@@ -170,6 +172,7 @@ func run(ctx context.Context) error {
 		}
 
 	case "TXT-NATIVE":
+		// FIXME: change this to use gql+dns lookup?
```
Comment (tqbf) [inline internal/command/dig/dig.go:175]: we probably don't need to merge API into TXT dig queries.

File: internal/command/machine/proxy.go [hunk 2]
```diff
@@ -75,7 +75,8 @@ func runMachineProxy(ctx context.Context) error {
 		Ports:            []string{"4280"},
 		OrganizationSlug: orgSlug,
 		Dialer:           dialer,
-		RemoteHost:       "_api.internal",
+		// FIXME: consider using fdaa::3 here
```
Comment (tqbf) [inline internal/command/machine/proxy.go:78]: Nah. There's no win to avoiding the _static_ DNS names, which aren't subject to propagation delays.

File: internal/command/postgres/attach.go [hunk 3]
```diff
@@ -127,10 +139,16 @@ func runAttach(ctx context.Context) error {
 		if err := hasRequiredVersionOnMachines(members, MinPostgresHaVersion, MinPostgresHaVersion); err != nil {
 			return err
 		}
+		for _, member := range members {
```
Comment (davissp14) [inline internal/command/postgres/attach.go:142]: 👍

Comment (davissp14) [reply internal/command/postgres/attach.go:142]: While this will work, we should consider just targeting the PG leader here.

**Review by davissp14 — APPROVED** [review]
LGTM

**Pull request merged**